test/e2e/landing-page.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { test, expect } from '@playwright/test';
+import { loggedInAsUser } from './utils';
+
+test.describe('landing page link', () => {
+  loggedInAsUser();
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/');
+
+    const dnsCard = page.getByRole('heading', { name: 'DNS Records' });
+    const certCard = page.getByRole('heading', { name: 'Certificate' });
+
+    await expect(dnsCard).toContainText('DNS Records');
+    await expect(certCard).toContainText('Certificate');
```

Comment:
This should be its own `test()` block, maybe something like 
```ts
test('contains DNS Record and Certificate cards', ...)
```

Reply:
I don't find that necessary. The purpose of that test.before is to check if it renders the landing page's cards before doing any of the test.

Reply:
Usually, you use before to do something that is required to test in the `test` blocks. This is kinda breaking away from that. Putting this in a `test` block would do the same thing

Reply:
Oh I guess that's true... Alright that make sense